### PR TITLE
hw3 function returning method by sector

### PR DESCRIPTION
**Pull request by MaVaYuran**

### CODE REVIEW

**Comment by MaVaYuran** [comment]
I have a question. if i add one more condition(if) for sector the first if doestn't work. why?

File: src/main/java/homework2/Ticket.java [hunk 1]
```diff
@@ -43,10 +43,12 @@ private String getTime() {
         return dateTime;
     }
 
-    public String getId() {
-        return id;
+    public Sector getSector() {
+        return sector;
     }
```
Comment (Alexmansar) [inline src/main/java/homework2/Ticket.java]: just create new method, this can be left. just getter))

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@MaVaYuran I small comment))
and, this task also shjild be mered to develop, so change 
image

File: src/main/java/homework2/Ticket.java (hunk 1 above)
Comment (MaVaYuran) [reply src/main/java/homework2/Ticket.java]: What method i should create? I dont
 understand

Comment (Alexmansar) [reply src/main/java/homework2/Ticket.java]: I this case you remove getId, and add getSector
Just add getSector without removing getId)

Comment (MaVaYuran) [reply src/main/java/homework2/Ticket.java]: You are my reviewer again)) Did you add yourself?

Comment (Alexmansar) [reply src/main/java/homework2/Ticket.java]: just automating gitHub after I left a comment.

**Review by Alexmansar — APPROVED** [review]
@MaVaYuran ypu can merge it. firstly, fix conflicts)) see lecture

**Comment by Alexmansar** [comment]
@MaVaYuran the question about if is still relevant?

**Comment by MaVaYuran** [comment]
about if alredy fixed)). But about merge, i should left both methods?

**Comment by Alexmansar** [comment]
Yes, both

**Pull request closed without merging**